src/main/flight/mixer.c

```diff
@@ -644,13 +644,15 @@ uint16_t convertExternalToMotor(uint16_t externalValue)
     uint16_t motorValue = externalValue;
 #ifdef USE_DSHOT
     if (isMotorProtocolDshot()) {
-        motorValue = externalValue <= EXTERNAL_CONVERSION_MIN_VALUE ? DSHOT_DISARM_COMMAND : constrain((externalValue - EXTERNAL_DSHOT_CONVERSION_OFFSET) * EXTERNAL_DSHOT_CONVERSION_FACTOR, DSHOT_MIN_THROTTLE, DSHOT_MAX_THROTTLE);
+        // Add 1 to the value, otherwise throttle tops out at 2046
+        motorValue = externalValue <= EXTERNAL_CONVERSION_MIN_VALUE ? DSHOT_DISARM_COMMAND : constrain((externalValue - EXTERNAL_DSHOT_CONVERSION_OFFSET) * EXTERNAL_DSHOT_CONVERSION_FACTOR + 1, DSHOT_MIN_THROTTLE, DSHOT_MAX_THROTTLE);
```

Comment:
Has this been tested over the full range of input values? I suspect this will cause runaway acceleration for some input values in configurator, since there is a feedback loop when actual motor values are converted by https://github.com/lazd/cleanflight/blob/06dba20efc6666d60df43748ef5033a45ea04d52/src/main/flight/mixer.c#L673, sent to the configurator, and then sent back to the flight controller and converted by `convertExternalToMotor`.